server/player/src/packet_handlers/placement.rs

```diff
@@ -215,12 +218,12 @@ fn handle_slab_placement(
     mut target_block_pos: BlockPosition,
     placement_face: PacketFace,
 ) -> bool {
-    if !block_to_place.is_slab() {
+    if !matches!(block_to_place.simplified_kind(), SimplifiedBlockKind::Slab) {
```

Comment:
`block_to_place.simplified_kind() == SimplifiedBlockKind::Slab` would be more succinct here. The same applies in a few more lines in this diff.

Reply:
Not sure why I was so hellbent on that macro :D